src/ngRoute/route.js

```diff
@@ -591,9 +591,8 @@ function $RouteProvider() {
                 if (angular.isFunction(templateUrl)) {
                   templateUrl = templateUrl(nextRoute.params);
                 }
-                templateUrl = $sce.getTrustedResourceUrl(templateUrl);
```

Comment:
- If the templateUrl is a string in the cache, then it is a behavior change, as it will not call getTrustedResourceUrl, but that's the point. 
- If the templateUrl is not a string or not in the cache, the call to valueOf below unwraps trusted types without side-effects, and we're just moving the call to getTrustedResourceUrl inside the call to templateRequest two lines below. You could maybe build a templateUrl equal to $sce.trustAs(SCE_CONTEXTS.RESOURCE_URL, undefined) which will get in the if block after the patch but not before, but the default $sce.trustAs implementation does not allow that, and it would only fail to load anyways.